src/customizations/components/theme/Unauthorized/Unauthorized.jsx

```diff
@@ -2,24 +2,74 @@
  * @module components/theme/Unauthorized/Unauthorized
  */
 
-import React from 'react';
-import { useLocation, Redirect } from 'react-router-dom';
+import React, { useEffect } from 'react';
+import { useLocation, Link, useHistory } from 'react-router-dom';
 import { getBaseUrl } from '@plone/volto/helpers';
+import { Container, Button } from 'semantic-ui-react';
+
+import { FormattedMessage } from 'react-intl';
+
+import { withServerErrorCode } from '@plone/volto/helpers/Utils/Utils';
 
 /**
  * unauthorized function.
  * @function Unauthorized
  * @returns {string} Markup of the unauthorized page.
  */
+
 const Unauthorized = () => {
-  let location = useLocation();
+  const location = useLocation();
+  const pathname = location.pathname;
+  const base_pathname = getBaseUrl(pathname);
+  const login = `${base_pathname}/login`;
+  const history = useHistory();
+  useEffect(() => {
+    const login_with_return = login + `?return_url=${pathname}`;
+    setTimeout(() => {
+      return history.push(login_with_return);
+    }, 10000);
+  }, [history, login, pathname]);
   return (
-    <Redirect
-      to={`${getBaseUrl(location.pathname)}/login?return_url=${
-        location.pathname
-      }`}
-    />
+    <Container className="view-wrapper">
+      <h1>
+        <FormattedMessage id="Unauthorized" defaultMessage="Unauthorized" />
+      </h1>
+      <h3>
+        <FormattedMessage
+          id="Forbidden, you will be redirected to the login screen in 10 seconds"
```

Comment:
* I think 10s is to much, 5s should be enough:
* This should be live / dynamic, like: 5, 4, 3, 2, 1